### PR TITLE
Admin panel backend mcp file (cloudstorage system file) deletion

### PR DESCRIPTION
**Pull request by timiimit**

### CODE REVIEW

File: UT4MasterServer.Services/Scoped/CloudstorageService.cs [hunk 1]
```diff
@@ -21,6 +21,23 @@ public sealed class CloudStorageService
...
 	private readonly ILogger<CloudStorageService> logger;
 
+	private static readonly string[] commonSystemFileFilenames = new string[]
+	{
+		// yes, 3 of these are misspelled by epic
+		"UnrealTournamentOnlineSettings.json",
+		"UnrealTournmentMCPAnnouncement.json",
+		"UnrealTournmentMCPGameRulesets.json",
+		"UnrealTournmentMCPStorage.json",
+		"UTMCPPlaylists.json"
```
Comment (Saibamen) [inline UT4MasterServer.Services/Scoped/CloudstorageService.cs:31]:
```suggestion
		"UTMCPPlaylists.json",
```

**Review by Saibamen — APPROVED** [review]

File: UT4MasterServer.Services/Scoped/CloudstorageService.cs (hunk 1 above)
Comment (timiimit) [reply UT4MasterServer.Services/Scoped/CloudstorageService.cs:31]: meh, the list will never change. don't think there's any point to this.

Comment (Saibamen) [reply UT4MasterServer.Services/Scoped/CloudstorageService.cs:31]: Just like `commonUserFileFilenames`, but you added a trailing comma there. I'm just saying that your new code in this same PR have inconsistent formatting

File: UT4MasterServer.Services/Scoped/CloudstorageService.cs [hunk 2]
```diff
@@ -21,6 +21,23 @@ public sealed class CloudStorageService
 	private readonly IMongoCollection<CloudFile> cloudStorageCollection;
 	private readonly ILogger<CloudStorageService> logger;
 
+	private static readonly string[] commonSystemFileFilenames = new string[]
```
Comment (belmirp) [inline UT4MasterServer.Services/Scoped/CloudstorageService.cs:24]: Use PascalCase.

File: UT4MasterServer.Services/Scoped/CloudstorageService.cs [hunk 3]
```diff
@@ -21,6 +21,23 @@ public sealed class CloudStorageService
...
+	{
+		// yes, 3 of these are misspelled by epic
+		"UnrealTournamentOnlineSettings.json",
+		"UnrealTournmentMCPAnnouncement.json",
+		"UnrealTournmentMCPGameRulesets.json",
+		"UnrealTournmentMCPStorage.json",
+		"UTMCPPlaylists.json"
+	};
+	private static readonly string[] commonUserFileFilenames = new string[]
```
Comment (belmirp) [inline UT4MasterServer.Services/Scoped/CloudstorageService.cs:33]: Use PascalCase.

**Review by belmirp — APPROVED** [review]

**Pull request merged**